### PR TITLE
Ensure the patient instructions are visible on the order card

### PR DESCRIPTION
**Pull request by mherman22**
## Requirements

 - [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
 - [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
 - [x] My work includes tests or is validated by existing tests.

## Summary

It was/is possible to fill in patient instructions in the drug order form however the patient instructions were not visible on the order card.

The designs can be found at https://zeroheight.com/23a080e38/p/030b9a-order-basket-item

## Screenshots

[Screencast from 15-02-23 12:50:12.webm](https://user-images.githubusercontent.com/47120265/218993603-f0f7efef-e595-497c-a0d7-2e82db1f93f0.webm)

## Related Issue

https://issues.openmrs.org/browse/O3-1845

### CODE REVIEW

**Comment by vasharma05** [comment]
Please add the updated screenshot.
I can't see the added instructions.

File: packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx [hunk 1]
```diff
@@ -56,7 +56,8 @@ export default function OrderBasketItemTile({ orderBasketItem, onItemClick, onRe
             &mdash; {orderBasketItem.route?.value} &mdash; {orderBasketItem.frequency?.value} &mdash;{' '}
             {t('refills', 'Refills').toUpperCase()} {orderBasketItem.numRefills}{' '}
             {t('quantity', 'Quantity').toUpperCase()}{' '}
-            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`}
+            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`} &mdash; {' '}
+            {`${orderBasketItem.patientInstructions}`}
```
Comment (vasharma05) [inline packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx:60]:
```suggestion
            {orderBasketItem.patientInstructions}
```

Comment (vasharma05) [inline packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx:60]: If the `patientInstructions` are not available, then the `mdash` will be left hanging. Please wrap the mdash properly.
```suggestion
            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`} 
            {orderBasketItem.patientInstructions && <>&mdash; {orderBasketItem.patientInstructions}</>}
```

Comment (vasharma05) [reply packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx:60]: Please wrap all the remaining mdashes properly too.
Thanks!

**Review by vasharma05 — CHANGES_REQUESTED** [review]

File: packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx (hunk 1 above)
Comment (mherman22) [reply packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx:60]: > Please wrap all the remaining mdashes properly too.
> Thanks!

is this an instruction for mdashes in the entire file?

**Review by vasharma05 — APPROVED** [review]
LGTM! Thanks @mherman22 !

**Comment by mherman22** [comment]
@vasharma05 i am tagging you just incase you forgot about this!

**Review by vasharma05 — APPROVED** [review]

**Comment by gracepotma** [comment]
Nice work @mherman22! Quick question for Vineet; shouldn't the blue bar at the top continue to be updated with the order details as the order is updated? Or is it behaving as you expect it to?

**Pull request merged**